lib/crate_ruby/version.rb

```diff
@@ -20,5 +20,5 @@
 # software solely pursuant to the terms of the relevant commercial agreement.
 
 module CrateRuby
-  VERSION = "0.0.7"
+  VERSION = "0.0.8"
```

Comment:
we usually bump a version with a separate commit on master
https://github.com/crate/crate_ruby/blob/master/DEVELOP.rst#preparing-a-release

Reply:
:+1: Reverted that commit.